cluster/gce/config-test.sh

```diff
@@ -551,3 +551,21 @@ GCE_PRIVATE_CLUSTER="${KUBE_GCE_PRIVATE_CLUSTER:-false}"
 ETCD_LISTEN_CLIENT_IP=0.0.0.0
 
 GCE_UPLOAD_KUBCONFIG_TO_MASTER_METADATA=true
+
+# ScaleOut arch vars
+PROXY_IMAGE_PROJECT=${PROXY_IMAGE_PROJECT:-ubuntu-os-cloud}
+PROXY_IMAGE=${PROXY_IMAGE:-ubuntu-1804-bionic-v20201014}
+PROXY_NAME="${SCALEOUT_PROXY_NAME:-scaleoutproxy}"
+PROXY_TAG="${SCALEOUT_PROXY_NAME:-scaleoutproxy}"
+
+# true if this is a resource partition cluster
+KUBERNETES_RESOURCE_PARTITION="${KUBERNETES_RESOURCE_PARTITION:-false}"
+# true if this is a tenant partition cluster
+KUBERNETES_TENANT_PARTITION="${KUBERNETES_TENANT_PARTITION:-false}"
+
+if [[ "${KUBERNETES_RESOURCE_PARTITION:-false}" == true ]]; then
```

Comment:
here too,  this will force the insecure port for TP or RP.

Reply:
fixed.